sklearn/multioutput.py

```diff
@@ -957,7 +989,7 @@ class labels for each estimator in the chain.
     ...    X, Y, random_state=0
     ... )
     >>> base_lr = LogisticRegression(solver='lbfgs', random_state=0)
-    >>> chain = ClassifierChain(base_lr, order='random', random_state=0)
+    >>> chain = ClassifierChain(estimator=base_lr, order='random', random_state=0)
```

Comment:
no need to pass the keyword, it should automatically use `estimator` when passed positionally